apps/openmw/mwgui/widgets.hpp

```diff
@@ -315,9 +315,6 @@ namespace MWGui
             virtual void initialiseOverride();
             void repeatClick(MyGUI::Widget* _widget, MyGUI::ControllerItem* _controller);
 
-            bool mEnableRepeat;
-            float mRepeatTriggerTime;
-            float mRepeatStepTime;
```

Comment:
These members were added to MyGUI::ScrollBar in 3.2.2. OpenMW still supports MyGUI 3.2.1 which doesn't have them. Ergo, we can't remove this code. Could add an #if guard, but tbh I'd prefer leaving everything as is.